src/particle.h

```diff
@@ -89,6 +95,32 @@ class Particle {
          * @return @b true if the Paticle is @b alive, false otherwise.
          */
         bool isAlive() const;
+
+        /**
+         * @brief Returns the position of the Particle.
+         */
+        ofPoint getPosition() const;
+
+        /**
+         * @brief Returns the color of the Particle.
+         */
+        ofColor getColor() const;
+
+        /**
+         * @brief Assigns a new color to the Particle.
+         *
+         * @param color -> The new color.
+         */
+        void setColor(const ofColor& color);
+
+        /**
+         * @brief Applies a force on the Particle (modifies its velocity).
+         *
+         * @param force The force to be applied.
+         *
+         * @param deltaTime The time in which the force is applied.
+         */
+        void applyForce(const ofPoint& force, float deltaTime);
```

Comment:
The application of the force should change only the acceleration. Treat the velocity update in updateParticle.